scripts/zones/RoMaeve/npcs/Goblin_Footprint.lua

```diff
@@ -0,0 +1,66 @@
+-----------------------------------
+-- Area: Bostaunieux Oubliette
+--  NPC: Goblin Footprint
+-- Type: NPC
+-- !pos  111.806 -24.007 57.266 167
+-----------------------------------
+local ID = require("scripts/zones/RoMaeve/IDs")
+require("scripts/globals/pasteventwatcher")
+require("scripts/globals/quests")
+require("scripts/globals/missions")
+-----------------------------------
+---
+function onTrade(player,npc,trade)
+    local options = {}
+    if player:hasCompletedMission(WINDURST,dsp.mission.id.windurst.VAIN) == true then
+        options[1] = 1
+    end
+    if player:hasCompletedMission(WINDURST,dsp.mission.id.windurst.MOON_READING) == true then
+        options[2] = 1
+    end
+    if player:getQuestStatus(ZILART,dsp.quest.id.outlands.DIVINE_MIGHT) == QUEST_COMPLETED then
+        options[3] = 1
+    end
+    if player:hasCompletedMission(ASA,dsp.mission.id.asa.FOUNTAIN_OF_TROUBLE) == true then
+        options[4] = 1
+    end
+    local eventList = dsp.pasteventwatcher.goblinFootprint({options[1], options[2], options[3], options[4]})
```

Comment:
Can't you just pass `options` here?

Reply:
Hi @nfaletra  I made the changes and now it looks nice  😸